index.css

```diff
@@ -2,4 +2,22 @@ h1 {
  color: red;
  font-family: 'Raleway', sans-serif;
  font-size: 30px;
```

Comment:
You can delete this code now, since it's part of the `Hello world` and it's not needed anymore.
```suggestion
```